### ISSUE TITLE
Help needed in understanding Saute output

### ISSUE THREAD

**Issue by swarnalilouha** [ncbi/SKESA#40]
I used Saute to assemble a reference fasta sequence '>CRYPT1020_1' from Illumina reads. I got 2 assemblies:
>CRYPT1020_1:1:1:87926 2 4 1
>CRYPT1020_1:1:2:87918 2 3 1

Why are there 2 assemblies and what do the numbers in the fasta headers mean?

**Comment by souvorov** [comment]
Often time assemblies have multiple variants. The simplest case are SNPs. SAUTE arranges all variants in a graph (output is controlled by --gfa option). You can analyze  this graph if you install BANDAGE (https://rrwick.github.io/Bandage/). Up to 1000 variants are printed by SAUTE in --all_variants in the fasta format. 
The first part of the fasta ID is Target name:graph number:contig number:estimated k-mer count. After that the numbers of the used graph nodes are printed separated by a space.
From what you posted one can say that your graph has two variants. The difference is represented by nodes 3 and 4. You should either look at the graph or align two contigs to understand what kind of difference they have.

**Comment by atongsa** [comment]
Can SAUTE be used to assemble whole genome sequencing (WGS) data for humans?

**Comment by atongsa** [comment]
yes, i not mean the whole genome, but only specific genes in human genome with SAUTE using human WGS

**Comment by souvorov** [comment]
Try the target sequences slightly exceeding the area of the gene of interest. It should work, unless there are large insertions/deletions/rearrangements inside the gene introns.

**Comment by atongsa** [comment]
thank you very much
